src/solvers/newtons_method.jl

```diff
@@ -2,6 +2,7 @@ export NewtonsMethod, KrylovMethod
 export JacobianFreeJVP, ForwardDiffJVP, ForwardDiffStepSize
 export ForwardDiffStepSize1, ForwardDiffStepSize2, ForwardDiffStepSize3
 export ForcingTerm, ConstantForcing, EisenstatWalkerForcing
+export KrylovMethodDebugger, PrintConditionNumber
 
```

Comment:
Let’s not export this for now

Reply:
Okay, should I also remove the documentation, or is that fine? Is the rule of thumb just "don't export things that will only used for debugging"?

Reply:
I'd like to push away from exporting things in general (to declutter the namespace when `using)`. Also, it makes no difference if we qualify methods, which we've been doing more of.